frontends/frontend-build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+
+QUAY_EXPIRE_TIME="${IMAGE_AUTO_DELETION_DAYS}d"
+
+# source "./helpers/general.sh"
+
+# trap_proxy teardown EXIT ERR SIGINT SIGTERM
+
+NODE_BUILD_VERSION=`node -e 'console.log(require("./package.json").engines.node.match(/(\d+)\.\d+\.\d+/)[1])'`
+
+IMAGE="$APP_IMAGE"
```

Comment:
The 'IMAGE` var is what's being used in the frontend-build.sh script from the [frontend common builder repo](                COMMON_BUILDER="https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master"). Unless we update that I had to change it so it can be used.